config/components/webhook/manifests.yaml

```diff
@@ -4,6 +4,25 @@ kind: MutatingWebhookConfiguration
 metadata:
   name: mutating-webhook-configuration
 webhooks:
+- admissionReviewVersions:
```

Comment:
Is this change required, or just formatting? If formatting, then I would suggest to revert.

Reply:
I believe it's automatically added on `make verify`.

Reply:
Hm, but it is only moved from another place. Why wasn't this then moved prior to this PR?

Reply:
I think it's because `_ "sigs.k8s.io/kueue/pkg/controller/jobs/deployment"` added  on `pkg/controller/jobs/jobs.go` and sort list changed on code-generator. In any case, `make verify` should throw an error if something not correct.

Reply:
sgtm, thanks!